### PR TITLE
[ManagerAccessControl]Add WooAccessManager for Manager

### PR DESCRIPTION
**Pull request by merlincdj**

### CODE REVIEW

File: contracts/WooFeeManager.sol [hunk 1]
```diff
@@ -66,18 +67,28 @@ contract WooFeeManager is InitializableOwnable, ReentrancyGuard, IWooFeeManager
     address public immutable quoteToken;
     IWooRebateManager public rebateManager;
     IWooVaultManager public vaultManager;
+    IWooAccessManager public wooAccessManager;
```
Comment (skyinthesea-woo) [inline contracts/WooFeeManager.sol:70]: accessManager.

**Review by skyinthesea-woo — CHANGES_REQUESTED** [review]
1. remove batch update and batch event
2. wooAccessManager --> accessManager
3. WooVaultManager, WooRebateManager also needs updated.

File: test/WooVaultManager.test.ts [hunk 2]
```diff
@@ -325,3 +330,129 @@ describe('WooVaultManager', () => {
...
+    await wooVaultManager.connect(owner).addReward(rewardAmount)
+    expect(await usdtToken.balanceOf(wooVaultManager.address)).to.eq(rewardAmount)
+
+    expect(await wooAccessManager.isVaultAdmin(user.address)).to.eq(false)
+    await expect(wooVaultManager.connect(user).distributeAllReward()).to.be.revertedWith(
+      onlyAdminRevertedMessage
+    )
+
+    expect(await wooAccessManager.isVaultAdmin(admin.address)).to.eq(true)
+    await wooVaultManager.connect(admin).distributeAllReward()
```
Comment (merlincdj) [inline test/WooVaultManager.test.ts:410]: 1) WooVaultManager Access Control
       Only admin able to distributeAllReward:
     Error: VM Exception while processing transaction: reverted with reason string 'TransferHelper::safeApprove: approve failed'
      at WooVaultManager.onlyAdmin (contracts/WooVaultManager.sol:85)
      at WooVaultManager.safeApprove (@uniswap/lib/contracts/libraries/TransferHelper.sol:7)
      at WooVaultManager.distributeAllReward (contracts/WooVaultManager.sol:154)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at runNextTicks (node:internal/process/task_queues:65:3)
      at listOnTimeout (node:internal/timers:526:9)
      at processTimers (node:internal/timers:500:7)
      at EthModule._estimateGasAction (node_modules/hardhat/src/internal/hardhat-network/provider/modules/eth.ts:425:7)
      at HardhatNetworkProvider.request (node_modules/hardhat/src/internal/hardhat-network/provider/provider.ts:108:18)
      at EthersProviderWrapper.send (node_modules/@nomiclabs/hardhat-ethers/src/internal/ethers-provider-wrapper.ts:13:20)

Comment (merlincdj) [reply test/WooVaultManager.test.ts:410]: fixed

**Review by skyinthesea-woo — APPROVED** [review]
Go！

**Pull request merged**